app/controllers/projects/blob_controller.rb

```diff
@@ -44,11 +50,16 @@ def update
     result = Files::UpdateService.new(@project, current_user, @commit_params).execute
 
     if result[:status] == :success
-      flash[:notice] = "Your changes have been successfully committed"
-      redirect_to after_edit_path
+      respond_to do |format|
+        format.html {redirect_to after_edit_path}
```

Comment:
I'm not sure how this passed Rubocop, but please format these lines as the following:

```ruby
format.html { redirect_to after_edit_path }
```